unbxdAutosuggest.js

```diff
@@ -1095,7 +1095,14 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, params) {
 				return "//search.unbxdapi.com/";
 			}
 			else {
-				return "//search.unbxd.io/";
+				if (
+					typeof this.options.searchEndPoint === 'string' &&
+					this.options.searchEndPoint.length > 0
+				  ) {
+					return this.options.searchEndPoint;
+				  } else {
+					return '//search.unbxd.io/';
```

Comment:
Instead let's add this in default config with default value '//search.unbxd.io/' and always return searchendpoint.

Reply:
Fixed